src/main/java/com/minecolonies/coremod/entity/EntityCitizen.java

```diff
@@ -248,8 +248,9 @@
     private ResourceLocation texture;
     private int              colonyId;
     private int citizenId = 0;
-    private int          level;
-    private int          textureId;
+    private int level;
+    private int textureId;
+    private final float volume = 0.2F;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this final field static too. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1170)